game_eggs/steamcmd_servers/counter_strike/counter_strike_2/egg-counter--strike2.json

```diff
@@ -0,0 +1,124 @@
+            "container": "ghcr.io\/parkervcp\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Map",
+            "description": "The default map for the server.",
+            "env_variable": "SRCDS_MAP",
+            "default_value": "de_dust2",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:de_dust2,de_mirage,de_nuke,de_overpass,de_inferno,de_ancient,de_vertigo,de_anubis,cs_italy,cs_office",
+            "field_type": "text"
+        },
+        {
+            "name": "Source AppID",
+            "description": "Required for game to update on server restart. Do not modify this.",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "730",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:730",
+            "field_type": "text"
+        },
+        {
+            "name": "Gamemode",
+            "description": "The default gamemode for the server.",
+            "env_variable": "GAMEMODE",
+            "default_value": "competitive",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|in:competitive,wingman,casual,deathmatch,custom",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "Specifies the maximum amount of players that are able to join the server.",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "12",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric|min:1|max:64",
```

Comment:
You can better hete use between.

Reply:
[73f47bf](https://github.com/parkervcp/eggs/pull/2725/commits/73f47bf0d2b231f577e34673e6267411706a96c0)